### PR TITLE
Implement projecting disp

### PR DESCRIPTION
**Pull request by MaxNoe**
This adds the possibility to train on projected disp instead of the distance between cog and source.
If delta (psi for ctapipe) is correct, this make now difference, on a test on FACT data, this showed no significant difference to the other method.

The two versions for a shower were delta is not exact:
true_disp

### CODE REVIEW

**Comment by LukasNickel** [comment]
The projected disp would be the end of the blue line in your picture while the "normal" disp would be slightly farther away from the cog, right?

**Comment by MaxNoe** [comment]
Projected disp is the length of the blue line, normal disp is the length of the gray line.

**Comment by LukasNickel** [comment]
> Projected disp is the length of the blue line, normal disp is the length of the gray line.

Right, my bad. I was referring to the resulting predictions. So with the imperfect delta in this case the normal disp prediction would be farther away than the projected one.

**Comment by MaxNoe** [comment]
This is for the training data, but yes. If the prediction of the respective disp would be perfect, the projected disp would be better in this case.

However, on the FACT data, it did not make a difference.

**Comment by LukasNickel** [comment]
Slightly off topic: Do you expect the error on the angle to be mainly random due to the limited camera resolution and cleaning effects or is there some systematic based on e.g. the position of the telescope(s) relative to the interaction point in the air?

File: aict_tools/parallel.py [hunk 1]
```diff
@@ -23,6 +24,7 @@ def parallelize_array_computation(func, *arrays, n_jobs=-1):
         end = start + block_size
         blocks.append([a[start:end] for a in arrays])
 
+    func = partial(func, **kwargs)
```
[the remark this answers is not in the document]
Comment (MaxNoe) [reply aict_tools/parallel.py]: This is used to pass `project_disp` in `fact_to_dl3`

**Pull request merged**